src/Widgets/TestimonialWidget/TestimonialWidgetEditingConfig.js

```diff
@@ -20,6 +19,38 @@ Scrivito.provideEditingConfig("TestimonialWidget", {
     author: "Someone",
     testimonial: "This is great!",
   },
-  titleForContent: widget =>
-    `${widget.get("author")}: ${truncate(widget.get("testimonial"))}`,
+  validations: [
+    [
+      "author",
+
+      author => {
+        if (!author) {
+          return {
+            message: "The author should be set.",
+            severity: "warning",
+          };
+        }
+      },
+    ],
+    [
+      "testimonial",
+
+      testimonial => {
+        if (!testimonial) {
+          return {
+            message: "The testimonial should be set.",
+            severity: "warning",
+          };
+        }
+      },
+    ],
+  ],
+  titleForContent: widget => {
+    const parts = [widget.get("author"), widget.get("testimonial")];
```

Comment:
Let's use truncate again to limit the length of the testimonial. Otherwise it can become quite long.

```suggestion
    const parts = [widget.get("author"), truncate(widget.get("testimonial"))];
```